resources/js/electron-builder.js

```diff
@@ -16,25 +16,44 @@ const isArm64 = process.argv.includes('--arm64');
 const isWindows = process.argv.includes('--win');
 const isLinux = process.argv.includes('--linux');
 const isDarwin = process.argv.includes('--mac');
-let targetOs = 'mac';
 let phpBinaryFilename = 'php';
-if (isWindows) {
+// Default to the current platform for develop mode. Build will pass in an arg that overrides this
+let targetOs = process.platform;
```

Comment:
Set the target to the current OS, if building, an override flag will be sent in.